widgets/card_widget.php

```diff
@@ -10,35 +10,35 @@
  *
  * @since 1.0.0
  */
-class Wasyo_Image_Card extends \Elementor\Widget_Base
+class Image_Card extends \Elementor\Widget_Base
```

Comment:
This is a poor name for the class. File names should reflect class names. A file named card_widget.php is expected to provide the class CardWidget. You should therefore rename the class to CardWidget.